app/models/ems_refresh/save_inventory_helper.rb

```diff
@@ -42,6 +42,7 @@ def save_inventory_multi(association, hashes, deletes, find_key, child_keys = []
       deletes.reload
     end
     deletes = deletes.to_a
+    deletes_index = deletes.index_by { |x| x }
```

Comment:
Hm, in this case, those 2 approach efficiency should be the same. Since Set seems to use Hash for include? and delete operations.

But I guess it's clearer for me to use Hash, rather then having Hash hidden behind Set. :-)

Btw. in DTO I don't use this, all the elements are already indexed by their unique keys.